assets/build/webpack.config.watch.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
       target: config.devUrl,
       proxyUrl: config.proxyUrl,
       watch: config.watch,
+      delay: 500,
```

Comment:
The delay is an arbitrary one set by my plugin to invoke a delay between local file system updates and injection/reload by BrowserSync. This is to resolve an issue with BrowserSync reloading before the server (i.e., VM) has synchronized with local file changes. My plugin by default has this value set to 50ms, but it's 500ms in Sage because I'm assuming that most developers are using a VM.

Anyway, I could add a separate option for debounce as well, or possibly refactor my current implementation to use debounce instead. I'll look into it and do some tests.